sound/utils.go

```diff
@@ -21,34 +21,31 @@ package sound
 
 import (
 	"fmt"
-	"os"
 	"path"
-	"pkg.deepin.io/lib/utils"
+	xdg "pkg.deepin.io/lib/xdg/basedir"
 )
 
 var (
 	errInvalidEvent = fmt.Errorf("invalid theme event")
 )
 
 func findThemeFile(theme, event string) (string, error) {
-	var home = os.Getenv("HOME")
-	var themeDirs = []string{
-		path.Join(home, ".local/share/sounds"),
-		"/usr/local/share/sounds",
-		"/usr/share/sounds",
+	// TODO: fix non ogg/oga event
+	// TODO: handle sound theme 'index.theme'
+	var pattern = path.Join("sounds", theme, "stereo", event, ".%s")
+	needle := fmt.Sprintf(pattern, "ogg")
+	file, err := xdg.FindFileInDataDirs(needle)
+
+	if err != nil {
```

Comment:
_Slaps forehead_

Of course, thank you! And sorry for the noise - I meant to open this against my own fork for now. But since it's already here - would you be willing to accept changes of this nature?